.github/workflows/main-build.yml

```diff
@@ -180,6 +181,10 @@ jobs:
           tags: |
             ${{ env.STAGING_ECR_REGISTRY }}/${{ env.STAGING_ECR_REPOSITORY }}:${{ steps.java_agent_versioning.outputs.STAGING_TAG }}
 
+      - name: Set image name to output
+        id: imageNameOutput
+        run: echo "imageName=${{ env.STAGING_ECR_REGISTRY }}/${{ env.STAGING_ECR_REPOSITORY }}:${{ steps.java_agent_versioning.outputs.STAGING_TAG }}" >> "$GITHUB_OUTPUT"
```

Comment:
Do we have these env variables added in repo

Reply:
yes the env variables are declared here: https://github.com/aws-observability/aws-otel-java-instrumentation/blob/2aed13f4c5d47f35f4f81a7157a73c930b68ed6d/.github/workflows/main-build.yml#L10

these are just global placeholder for the image tag.